src/Core/Models/Business/Provider/ProviderUserInvite.cs

```diff
@@ -1,19 +1,15 @@
+using System;
 using System.Collections.Generic;
 using Bit.Core.Enums.Provider;
 using Bit.Core.Models.Api;
-using Bit.Core.Models.Data;
 
 namespace Bit.Core.Models.Business.Provider
 {
-    public class ProviderUserInvite
+    public class ProviderUserInvite<T>
     {
-        public IEnumerable<string> Emails { get; set; }
+        public IEnumerable<T> UserIdentifiers { get; set; }
```

Comment:
this is nice. I think it's quite clear that strings will send new invites and guids will be reinvites.

I wish we could limit to those types...